logo.js

```diff
@@ -1600,10 +1738,12 @@ function LogoInterpreter(turtle, stream, savehook)
   def("true", function() { return 1; });
   def("false", function() { return 0; });
 
+// FIXME: needs to handle promises
```

Comment:
Hrm... seems like the caller should handle deferring until the inputs are settled, so these procedures never see promises...